src/main/java/cloud/fogbow/ras/constants/Messages.java

```diff
@@ -97,6 +97,7 @@
         public static final String CONNECTING_UP_PACKET_SENDER = "Connecting XMPP packet sender.";
         public static final String DELETING_INSTANCE = "Deleting instance %s with token %s.";
         public static final String DELETING_INSTANCE_S = "Deleting instance %s.";
+        public static final String END_DELETE_NIC_ASYNC_BEHAVIOUR = "End asynchronous delete network interface";
```

Comment:
Add the "." at the end of the text in order to follow the pattern.

Reply:
Fixed.